src/BenchmarkDotNet/Toolchains/Mono/MonoGenerator.cs

```diff
@@ -0,0 +1,19 @@
+﻿using BenchmarkDotNet.Characteristics;
+using BenchmarkDotNet.Jobs;
+using BenchmarkDotNet.Toolchains.CsProj;
+
+namespace BenchmarkDotNet.Toolchains.Mono
+{
+    public class MonoGenerator : CsProjGenerator
+    {
+        public MonoGenerator(string targetFrameworkMoniker, string cliPath, string packagesPath, string runtimeFrameworkVersion, bool isNetCore = true) : base(targetFrameworkMoniker, cliPath, packagesPath, runtimeFrameworkVersion, isNetCore)
```

Comment:
nit: `isNetCore` will always be true for this generator

```suggestion
        public MonoGenerator(string targetFrameworkMoniker, string cliPath, string packagesPath, string runtimeFrameworkVersion) : base(targetFrameworkMoniker, cliPath, packagesPath, runtimeFrameworkVersion, true)
```